src/StakeRegistry.sol

```diff
@@ -491,7 +491,8 @@ contract StakeRegistry is StakeRegistryStorage {
         uint256 stratsLength = strategyParamsLength(quorumNumber);
         StrategyParams memory strategyAndMultiplier;
 
-        uint256[] memory strategyShares = delegation.getOperatorShares(operator, strategiesPerQuorum[quorumNumber]);
+        uint256[] memory strategyShares;
+        //  = delegation.getDelegatableShares(operator, strategiesPerQuorum[quorumNumber]);
```

Comment:
?

Reply:
The interface changed just commented out temporarily. It was initially more annoying to deal with because the type returned changed and then the function signature changed

Reply:
Resolved in [e08cc44](https://github.com/Layr-Labs/eigenlayer-middleware/pull/310/commits/e08cc4465595aea3b29bc73cd47829c2041cb523)